eslint.config.mjs

```diff
@@ -0,0 +1,24 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+export default [
+  {
+    ignores: [
+      'dist'
+    ]
+  },
+  ...bpmnIoPlugin.configs.recommended,
+  ...bpmnIoPlugin.configs.mocha.map(config => {
+    return {
+      ...config,
+      files: [
+        'test/**/*.js'
+      ],
+      languageOptions: {
```

Comment:
From my point of view better: 

Just have a separate configuration with "additional test changes" to be merged with the previosu config, cf. https://github.com/bpmn-io/diagram-js/pull/936/commits/85027fef783bf52e20cd49d94d8f36a24a26ff39

:arrow_right: ESLint does the merging already, I don't see why we need to do it.

Reply:
Yeah

Reply:
https://github.com/bpmn-io/dmn-migrate/commit/0489d45a033f4f4aa5d7159e2407b3172de76a41